app/models/restaurant.rb

```diff
@@ -0,0 +1,31 @@
+# == Schema Information
+#
+# Table name: restaurants
+#
+#  id                :integer          not null, primary key
+#  name              :string
+#  accepts_10_bis    :boolean
+#  address           :string
+#  max_delivery_time :integer
+#  longitude         :float
+#  latitude          :float
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#
+
+class Restaurant < ApplicationRecord
+  has_one :cuisine
+  has_many :reviews
+
+  def as_json(*)
```

Comment:
Have you considered not having this in the model?

Reply:
I would love to get your input on how this should be done, i fiddled with this a little and couldn't come up with something better.